docs/build/build-hrmp-channels.md

```diff
@@ -7,40 +7,56 @@ keywords: [HRMP, parachain, statemint, proposal, asset hub, statemine]
 slug: ../build-hrmp-channels
 ---
 
-[HRMP](../learn/learn-xcm.md#xcmp-lite-hrmp) has the same interface and functionality as
-[XCMP](../learn/learn-xcm.md#xcmp-cross-chain-message-passing) but is much more demanding on
-resources since it passes all messages via the Relay Chain. When XCMP is implemented on Polkadot,
+[HRMP](../learn/learn-xcm-transport.md#hrmp-xcmp-lite) has the same interface and functionality as
+[XCMP](../learn/learn-xcm-transport.md#xcmp-cross-chain-message-passing) but is much more demanding
+on resources since it passes all messages via the Relay Chain. When XCMP is implemented on Polkadot,
 HRMP is planned to be deprecated and phased out in favor of it.
 
 ## Opening HRMP channel: parachain to parachain
 
-To open a channel from one parachain to another that are not system chains on Polkadot, the
-governance of each parachain needs to declare its intent to open a channel on the Relay Chain, and
+To open a channel from one parachain to another, that is not system chains on Polkadot, the
+governance of each parachain needs to declare its intent to open a channel on the Relay Chain and
 the second chain needs to accept and do the same.
 
-In order to do this, the calls to be dispatched from both chains are:
+To do this, the calls to be dispatched from both chains are:
 
 1. `hrmp > hrmpInitOpenChannel(recipient, proposedMaxCapacity, proposedMaxMessageSize)`: this call
-   initiates opening a channel from a parachain to a given recipient with given channel
+   initiates opening a channel from a parachain to a given recipient.
 
-2. `hrmp > hrmpAcceptOpenChannel(sender)`: accepting the channel open request from the given sender.
+2. `hrmp > hrmpAcceptOpenChannel(sender)`: this call is for accepting the channel open request from
+   the given sender.
 
-Each parachain is to use its own governance process to execute this. The call will be executed on
-the Relay Chain.
+Each parachain is to use its governance process to execute this. The call will be executed on the
+Relay Chain.
```

Comment:
If it is to be made only through governance, then why do we have the xcm instruction `[HrmpNewChannelOpenRequest](https://paritytech.github.io/xcm-docs/journey/channels-and-bridges.html#hrmpnewchannelopenrequest)` sent by the relay as a notification?
At Polimec we are using this to do automatic channel opening, without going through governance.

Reply:
Although we had to modify the xcm-executor because it had that instruction left empty, so maybe that change can happen once the executor adds an implementation for those instructions 🤔